django/db/backends/sqlite3/schema.py

```diff
@@ -558,3 +572,12 @@ def remove_constraint(self, model, constraint):
 
     def _collate_sql(self, collation):
         return "COLLATE " + collation
+
+    def _format_sql_for_default(self, wrapper_sql, default_sql, params):
+        # Sqlite does not support parametrized database defaults
+        # so format here with quote_value instead.
+        default_sql = default_sql.replace("%s", "{}")
+        default_sql = default_sql.format(
+            *tuple(self.quote_value(value) for value in params)
+        )
```

Comment:
This is really ugly, but it avoids python getting confused by other percent formatting that could be present in `default_sql`, for example the `%Y-%m-%d %H:%M:%f` formatting for `Now()` introduced in #16111.

The other fix is to escape those formatting characters an extra level, but I'm not sure that's better.

Reply:
I'm open to other ideas!

Reply:
> This is really ugly, but it avoids python getting confused by other percent formatting that could be present in `default_sql`, for example the `%Y-%m-%d %H:%M:%f` formatting for `Now()` introduced in #16111.

It was corrected in #16223.